learn_php/casting.php

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html>
+    <body>
+        <?php 
+        /*To cast to string, use the (string) statement:*/ 
```

Comment:
Nếu comment 1 line, thì dùng như bên dưới
```suggestion
        // To cast to string, use the (string) statement:
```
Nếu comment nhiều line thì như bên dưới
```php
    /**
     * Set extra property
     * Use to set value of property (type ID) from $_GET
     */
```

Reply:
Sửa toàn bộ những vị trí khác.